textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/source/SampleDataStoreTest.java

```diff
@@ -0,0 +1,27 @@
+/**

```

Comment:
Currently RegexMatcherTest tests ScanBasedSourceOperator class also. But that is not unit testing. Will add Unit test cases for ScanBasedSourceOperator.